sources/packages/web/src/components/institutions/locations/LocationEditForm.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <formio
+    formName="institutionlocation"
+    :data="locationData"
+    @submitted="updateInstitutionLocation"
+  ></formio>
+</template>
+
+<script lang="ts">
+import formio from "@/components/generic/formio.vue";
```

Comment:
you can ignore this since you are already importing in sources/packages/web/src/main.ts